nautobot/docs/user-guide/administration/tools/nautobot-server.md

```diff
@@ -70,6 +70,18 @@ If your filter data is valid, you should see a success message at the end of the
 >>> All DynamicGroup filters are validated successfully!
 ```
 
+### `build_ui`
+
+`nautobot-serve build_ui`
+
+Build the UI for the Nautobot server environment and installed Nautobot Apps.
```

Comment:
```suggestion
Build or rebuild the UI for the Nautobot server environment and installed Nautobot Apps. This command is normally run as part of [`nautobot-server post_upgrade`](#post_upgrade) rather than being invoked directly by an administrator.
```